cmd/dagger/span_name_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSpanName(t *testing.T) {
+	type example struct {
+		name string
+		args []string
+		want string
+	}
+	for _, test := range []example{
+		{args: []string{"dagger", "version"}, want: "dagger version"},
+		{args: []string{"dagger", "install", "foo"}, want: "dagger install foo"},
+		{args: []string{"dagger", "call", "foo"}, want: "foo"},
+		{args: []string{"dagger", "-m", "dev", "call", "foo"}, want: "foo"},
+		{args: []string{"dagger", "call", "-m", "dev", "foo"}, want: "foo"},
+		{
+			args: []string{"dagger", "call", "test", "important", "--race=true", "--parallel=16"},
+			want: "test important --race=true --parallel=16",
+		},
+		{args: []string{"dagger", "call", "--source", ".:default", "foo"}, want: "foo"},
+		{args: []string{"dagger", "call", "--source", ".:default", "foo", "--fizz"}, want: "foo --fizz"},
```

Comment:
suggestion: a testcase for `--source=.:default` combined, I think this is handled by the code above.